modules/op-constructing-pipelines-using-pipeline-builder.adoc

```diff
@@ -5,41 +5,53 @@
 [id="op-constructing-pipelines-using-pipeline-builder_{context}"]
 = Constructing Pipelines using the Pipeline Builder
 
-In the *Developer* perspective of the console, you can use the *Add* -> *Pipeline* -> *Pipeline Builder* option to:
+In the *Developer* perspective of the console, you can use the *+Add* -> *Pipeline* -> *Pipeline Builder* option to:
 
-* Construct a Pipeline flow using existing Tasks and ClusterTasks. When you install the OpenShift Pipelines Operator, it adds reusable Pipeline ClusterTasks to your cluster.
-* Specify the type of resources required for the Pipeline Run, and if required, add additional parameters to the Pipeline.
-* Reference these Pipeline resources in each of the Tasks in the Pipeline as input and output resources.
-* The parameters for a Task are prepopulated based on the specifications of the Task. If required, reference any additional parameters added to the Pipeline in the Task.
+* Configure pipelines using either the *Pipeline builder* or the *YAML view*.
+* Construct a pipeline flow using existing tasks and cluster tasks. When you install the OpenShift Pipelines Operator, it adds reusable pipeline cluster tasks to your cluster.
+* Specify the type of resources required for the pipeline run, and if required, add additional parameters to the pipeline.
+* Reference these pipeline resources in each of the tasks in the pipeline as input and output resources.
+* The parameters for a task are prepopulated based on the specifications of the task. If required, reference any additional parameters added to the pipeline in the task.
+* Use the operator-installed, reusable snippets and samples to create detailed pipelines.
 
 .Procedure
 
-. In the *Add* view of the *Developer* perspective, click the *Pipeline* tile to see the *Pipeline Builder* page.
-. Enter a unique name for the Pipeline.
-. Select a Task from the *Select task* list to add a Task to the Pipeline. This example uses the *s2i-nodejs* Task.
-.. To add sequential Tasks to the Pipeline, click the plus icon to the right or left of the Task, and from the *Select task* list, select the Task you want to add to the Pipeline. For this example, use the plus icon to the right of the *s2i-nodejs* Task to add an *openshift-client* Task.
-.. To add a parallel Task to the existing Task, click the plus icon displayed below the Task, and from the *Select Task* list, select the parallel Task you want to add to the Pipeline.
+. In the *+Add* view of the *Developer* perspective, click the *Pipeline* tile to see the *Pipeline Builder* page.
+. Configure the pipeline using either the *Pipeline builder* view or the *YAML view*.
++
+[NOTE]
+====
+The *Pipeline builder* view has limited supported fields whereas the *YAML view* has all the options. You can also use the operator-installed, reusable snippets and samples to create detailed Pipelines.
```

Comment:
We can add this sentence in the end like how we had it earlier: Optionally, you can also use the operator-installed, reusable snippets and samples to create detailed Pipelines.

Reply:
Done